### DISCUSSION TITLE
Importing a recipe to another tenant from Default tenant

### DISCUSSION THREAD

**Discussion by msquare98** [OrchardCMS/OrchardCore#10690] [General]
Is there a way to import recipe to other tenant while using Import feature in Deafult tenant. Is it possible to change the tenant(Shell settings) while executing recipe? I want to develop a feature to import recipe for all my tenants while i am using a single tenant. Any ideas how to change the tenant or shell settings?

**Comment by Piedone** [comment]
You'll need to create a new, custom recipe step for this, from which you [access the other tenant's services](https://orcharddojo.net/blog/how-to-access-services-from-another-tenant-in-orchard-core-orchard-nuggets).

An alternative is to [log in to the other tenant](https://github.com/Lombiq/Hosting-Tenants/blob/dev/Lombiq.Hosting.Tenants.Admin.Login/Readme.md) from the Default one, and just do the import there.
